tests/qactivation_test.py

```diff
@@ -417,34 +417,44 @@ def test_stochastic_binary():
   assert_allclose(scale, expected_scale, rtol=0.1)
 
 
-def test_stochastic_ternary():
+@pytest.mark.parametrize(
+    'alpha, temperature, expected_values, expected_scale', [
+    (
+        None, 
+        8,
+        np.array([0., 0., 0., 0., 0., 0., 0., 0., 0., 0.]).astype(np.float32),
```

Comment:
case seems a little "easy", 
could you add more with -alpha, 0, +alpha, where alpha can be 1, or other numbers?
also different scales as well.

Reply:
Added these on the latest commit

Reply:
thanks. to be more clear, I mean the value range between [-alpha, alpha].